packages/graphql/src/schema/make-augmented-schema.test.ts

```diff
@@ -152,29 +134,6 @@ describe("makeAugmentedSchema", () => {
         expect(() => makeAugmentedSchema({ typeDefs })).toThrow("cannot auto-generate an array");
     });
 
-    /*
-        Removal of validateTypeDefs function
-    */
-    // test("should throw timestamp operations must be an array", () => {
```

Comment:
I would say both of these test cases are valid and should be left in! The more coverage the better!

Reply:
These tests don't throw anymore.

Reply:
If that's the case, are we sure that this is validating against our directive definitions correctly?

Reply:
Yeah here https://github.com/neo4j/graphql/pull/232/files#diff-1332c2df9bfa483fbae8bd4af0e5b04092e34ad090756e1b895845c04e2d4c2dR28

Reply:
What I'm trying to get at is, if these tests don't throw any more, that means that the type of the `operations` field of the `@timestamp` directive isn't being validated against, and it's letting any values through. Unless I'm misunderstanding?

Reply:
It works fine and the GraphQL validate function will take care of it. As you can see I have asserted that we are hitting the GraphQL validation with my attached test and I don't think it's maintainable or necessary to assert the validation of each and every scalar, directive and type applied to the schema.